Sources/Purchasing/ProductRequestData.swift

```diff
@@ -20,6 +20,20 @@ import StoreKit
 /// - SeeAlso: `Backend/post(receiptData:appUserID:isRestore:productData:...`
 struct ProductRequestData {
 
+    /// Determins what triggered a receipt to be posted
+    enum InitiationSource: CaseIterable {
```

Comment:
Hmm what do you think about `ReceiptPostSource`? Feels like a better name to me but looks like renaming would require a backend change and I don't think it's a big deal.

Reply:
We don't have to match the backend, but I can see either way.
If we don't, we can have something more "Swifty", but I think the advantage of having this match the backend is familiarity, so this term is the same across different repos.
What do you think?

Reply:
Maybe another thing to realize is that this isn't just `InitiationSource`, it's namespaced as `ProductRequestData.InitiationSource`.